src/main/pages/che-7/hands-on-guidance/proc_configuring-a-che-workspace-using-a-devfile.adoc

```diff
@@ -1,27 +1,88 @@
 
 [discrete]
 == Prerequisites
 
-* A bulleted list of conditions that must be satisfied before the user starts following this assembly.
-* You can also link to other modules or assemblies the user must follow before starting this assembly.
-* Delete the section title and bullets if the assembly has no prerequisites.
-
+* You need an OKD, OCP, or OSD cluster, if using Che on Openshift. See below for use of link:https://www.okd.io/minishift/[Minishift].
+* Or, you need a Kubernetes cluster such as link:https://github.com/kubernetes/minikube#installation[Minikube] to use Che on Kubernetes.
+* For Che on OpenShift, an OKD, OCP, or OSD cluster. To use Minishift, see https://www.okd.io/minishift/. 
+* For Che on Kubernetes, a Kubernetes cluster like link:https://github.com/kubernetes/minikube#installation[Minikube].
 [discrete]
 == Procedure
 
-. Start each step with an active verb.
+=== Importing a project and workspace in one step using a devfile
+
+In Che 7, stacks are defined as link:using-developer-environments-workspaces.html#making-a-workspace-portable-using-a-devfile_using-developer-environments-workspaces[devfiles], instead of container images. This affords more flexibility when defining the project(s), IDE, commands, tools, and application runtimes needed in your workspace.
+
+So where in the past you might have included a `Dockerfile` in your project's repo, now you can include a `devfile.yaml` file in the root of your git project instead.
+
+==== The minimal devfile
+
+The minimum content you need in a `devfile.yaml` file is:
+
+* link:https://redhat-developer.github.io/devfile/devfile#apiversion[apiVersion]
+* link:https://redhat-developer.github.io/devfile/devfile#metadata[metadata name]
+* link:https://redhat-developer.github.io/devfile/devfile#projects[projects name and source]
+
+```
+apiVersion: 1.0.0
+metadata:
+  name: che-in-che-out
+projects:
+  - name: che
+    source:
+      type: git
+      location: 'https://github.com/eclipse/che.git'
+```
+
+For a more complete example, see link:https://github.com/eclipse/che/blob/master/devfile.yaml[Eclipse Che in Che devfile.yaml].
+
+
+==== Creating a workspace from a devfile
 
-. Include one command or action per step.
+. Commit a `devfile.yaml` into the root of your git repo.
 
-. Use an unnumbered bullet (*) if the procedure includes only one step.
+. Once your repo contains a `devfile.yaml`, point a running Che 7 instance at that link:https://github.com/eclipse/che[project] and it will use the discovered link:https://github.com/eclipse/che/blob/master/devfile.yaml[devfile.yaml] to build you a workspace via its Factory API, `/f?url=`.
```

Comment:
You made that way more confusing. 
New copy:

```
. Commit a `devfile.yaml` file into the root of your git repo.

. When your repository contains a `devfile.yaml` file, point a running Che 7 instance at that link:https://github.com/eclipse/che[project] and it will use the discovered link:https://github.com/eclipse/che/blob/master/devfile.yaml[devfile.yaml] file to build a workspace using the xref:api-reference#factory-api_api-reference[Factory API], `/f?url=`.
```

If you're going to remove the "your" references in one of the bullets, you should also remove it from the others... but as you didn't change "your git repo" or "if you're using..." I figure we should be consistent and keep all three in 2nd person voice.